### PR TITLE
Adds Ponies

### PR DESCRIPTION
**Pull request by optimumtact**
Adds ponies to the master branch

### CODE REVIEW

**Comment by KorPhaeron** [comment]
Much better, will merge as soon as possible

File: code/modules/mob/living/carbon/human/update_icons.dm [hunk 1]
```diff
@@ -62,6 +62,9 @@ Please contact me on #coderbus IRC. ~Carnie x
 /mob/living/carbon/human/update_damage_overlays()
 	remove_overlay(DAMAGE_LAYER)
 
+	if(dna && istype(dna.species, /datum/species/pony))
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/update_icons.dm]: these need to be replaced by generic variables on species.

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 2]
```diff
@@ -779,3 +779,188 @@ SYNDICATE BLACK OPS
...
+	var/pony_hair = null
+	var/pony_tail = null
+	var/alicorn_color = null
+	var/list/pony_types = list(
+		"rarity", "fleur", "twilight",
+		"pinkie", "lyra", "vinyl",
+		"whooves", "rainbow", "fluttershy")
+
+/datum/species/pony/New()
+	if(default_features["alicorn"] == "None" && prob(15))
```
Comment (Shadowlight213) [inline code/modules/mob/living/carbon/human/species_types.dm]: This shouldnt randomly set itself. It fucks with the character setup screen image.

**Comment by PKPenguin321** [comment]
Wew

**Comment by PKPenguin321** [comment]
Double wew

**Comment by Shadowlight213** [comment]
Damnit oranges. How am I supposed to meme merge now?

**Pull request closed without merging**